honestcash-v2/src/karma.conf.js

```diff
@@ -24,8 +24,8 @@ module.exports = function (config) {
     port: 9876,
     colors: true,
     logLevel: config.LOG_INFO,
-    autoWatch: false,
+    autoWatch: true,
```

Comment:
these will be reverted after I'm done with coding

Reply:
Can we ensure btw. that the tests run on Travis? This was part of this.